app/notes/notes.service.js

```diff
@@ -1,69 +1,75 @@
 (function() {
-  angular.module('meganote.notes')
+  angular
+    .module('meganote.notes')
     .service('NotesService', NotesService);
 
   NotesService.$inject = ['$http'];
   function NotesService($http) {
-    var service = this;
-    service.notes = [];
+    var vm = this;
```

Comment:
This is a service rather than a controller so you probably shouldv'e left `var service = this` since the service is not responsible for the view and would not need vm